### DISCUSSION TITLE
Cache statistics ?

### DISCUSSION THREAD

**Discussion by c-michail** [TechnitiumSoftware/DnsServer#1017] [General]
Hello,
  Is there any way to see how many records cache is holding and/or any other statistics ?
  
  Thank you

**Comment by ShreyasZare** [comment]
Thanks for asking. The small "stats bar" above the Top Clients on the Dashboard shows the number of entries in the Cache. Using this value, you can decide upon the Cache Maximum Entries option in Settings > Cache section.

The cached records also hold meta data which gives more details on the upstream server and the RTT.

**Reply by c-michail** [reply]
Awesome thank you!

**Reply by ShreyasZare** [reply]
You're welcome!
